prow/cmd/tide/config.md

```diff
@@ -0,0 +1,158 @@
+# Configuring Tide
+
+Configuration of Tide is located under the [prow/config.yaml](/prow/config.yaml) file. All configuration for merge behavior and criteria belongs in the `tide` yaml struct, but it may be necessary to also configure presubmits for Tide to run against PRs (see ['Configuring Presubmit Jobs'](#configuring-presubmit-jobs) below).
+
+Tide is configured by populating the following YAML structure. Depending on your criteria, some of the fields may by omitted.
+
+```yaml
+tide:
+  queries:                // List of queries
+  - orgs:                   // List of orgs
+    repos:                  // List of org/repo
+    labels:                 // List of must have labels
+    missingLabels:          // List of can't have labels
+    excludedBranches:       // Ignore branches
+    includedBranches:       // Include-only branches
+    reviewApprovedRequired: // Github Review approval required
+  sync_period:            // Sync jobs period
+  status_update_period:   // PR status update period
+  merge_method:           // Set to squash, rebase or just merge PR
+  target_url:             // URL linked to from the details link on the Github status context
+  pr_status_base_url:     // PR status page
+  max_goroutines:         // Per-pool parallelism
+```
+
+See [https://godoc.org/github.com/kubernetes/test-infra/prow/config#Tide](https://godoc.org/github.com/kubernetes/test-infra/prow/config#Tide) for the most up to date configuration specification.
```

Comment:
Hrm, especially since you have much more indepth doc below, I'm not even sure if the YAML snippet is useful.

Reply:
Yeah, I'll remove it. There is an example YAML config below that is probably easier to read and more useful anyways.